Apps/WebClient/src/ClientApp/test/reports.test.ts

```diff
@@ -17,13 +17,16 @@ import { RegistrationStatus } from "@/constants/registrationStatus";
 import { WebClientConfiguration } from "@/models/configData";
 import { DateWrapper } from "@/models/dateWrapper";
 import MedicationStatementHistory from "@/models/medicationStatementHistory";
+import { SnowplowWindow } from "@/plugins/extensions";
 import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
 import container from "@/plugins/inversify.container";
 import { ClientModule } from "@/router";
 import { ILogger } from "@/services/interfaces";
 import { GatewayStoreOptions } from "@/store/types";
 import ReportsView from "@/views/reports.vue";
 
+declare let window: SnowplowWindow;
```

Comment:
Is this doing anything?

Reply:
I think is needed to get the tests working. But I will double-check once it builds in dev